reactDashboard/src/components/DataCharts/DataCharts.css

```diff
@@ -58,43 +76,58 @@
 .region-chart-container {
   height: 100%;
   width: 100%;
-  background-color: #ffffff;
+  background-color: white;
 }
 
 .sonification-play-button {
   position: absolute;
   top: 20px;
   right: 40px;
-  background-color: #ffffff;
+  background-color: white;
   font-size: 0.5rem;
   border: 1px solid #6c6c6c;
   border-radius: 10px;
   padding: 7px;
   width: 7em;
   height: 7em;
+  color: black;
+}
+
+.sonification-play-button {
+  background-color: inherit;
+  border-color: #cccccc;
+}
+
+.sonification-play-button h5 {
+  font-size: 0.75rem;
+  color: #6c6c6c;
+  margin-bottom: 0;
+}
+
+.darkmode .sonification-play-button h5 {
+  color: #f2f2f2;
 }
 
 .sonification-play-button:hover {
   box-shadow: 5px 5px 5px #aaaaaa;
 }
 
+.darkmode .sonification-play-button:hover,
+.darkmode .sonification-play-button h5:hover {
```

Comment:
Do you need the directive for the h5 ? Wouldn't have expected there to be a box-shadow there.

Reply:
✅